Sources/StytchCore/StytchClientCommon/Models/Response.swift

```diff
@@ -28,6 +31,7 @@ public struct Response<Wrapped: Decodable>: Decodable {
     }
 
     /// Enables dynamic member access to the wrapped type.
+    /// Example usage: response[dynamicMember: \Type.property]
```

Comment:
Though this is a valid usage, its intention (and perhaps a more idiomatic example to include in the docs) was to take advantage of [dynamicMemberLookup](https://github.com/apple/swift-evolution/blob/main/proposals/0195-dynamic-member-lookup.md), where you could use `response.property` instead of explicitly utilizing the subscript. (👋 hi there and nice to meet you btw, just doing a random drop by to see how this codebase is coming along)